src/entities/purchase-flow-params.ts

```diff
@@ -0,0 +1,24 @@
+import { type Package, type PurchaseOption } from "./offerings";
+
+/**
+ * Parameters used to customise the purchase flow when invoking the `.purchase` method.
+ * @public
+ */
+export interface PurchaseFlowParams {
+  /**
+   * The package you want to purchase. Obtained from {@link Purchases.getOfferings}.
+   */
+  rcPackage: Package;
+  /**
+   * The option to be used for this purchase. If not specified the default one will be used.
+   */
+  purchaseOption?: PurchaseOption | null;
+  /**
+   * The HTML element where the billing view should be added. If null, a new div will be created at the root of the page and appended to the body.
+   */
+  htmlTarget?: HTMLElement;
+  /**
+   * The email of the user. If null, RevenueCat will ask the customer for their email.
```

Comment:
Same here, take it with a grain of salt since I am no expert, but I think this should be "If undefined" instead of "If null"